outplacement_final_report/__manifest__.py

```diff
@@ -1,6 +1,6 @@
 {
     'name': 'Outplacement final report',
-    'version': '12.0.0.2.7',
+    'version': '12.0.1.0.0',
```

Comment:
I don't see any other changes to this module, is updating the version number and readme actually relevant?